src/main/java/org/fundacionjala/coding/christian/Kata1.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.christian;
+
+import java.util.StringJoiner;
+
+/**
+ * Powered by Christian.
+ */
+
+public class Kata1 {
+
+    public static final int LIMIT = 5;
+
+    /**
+     * @param words this is words for Spin Words.
+     * @return result.
+     */
+    public String spinWords(String words) {
+
+        StringJoiner join = new StringJoiner(" ");
+        String[] arr = words.split(" ");
+        for (String c : arr) {
+            join = c.length() >= LIMIT
+                    ? join.add(new StringBuilder(c).reverse()) : join.add(c);
+        }
+
+        return join.toString();
```

Comment:
This can be refactor something like this.
```java
StringJoiner join = new StringJoiner(BLANK_SPACE);
        String[] arr = words.split(BLANK_SPACE);
        for (String c : arr) {
            join.add(c.length() >= LIMIT ? new StringBuilder(c).reverse() : c);
        }
        return join.toString();
```